content/300-guides/300-upgrade-guides/200-upgrading-versions/index.mdx

```diff
@@ -28,6 +28,7 @@ To upgrade to the latest version of Prisma:
 
 ### Prisma 2 and onwards
 
+- [Upgrade to Prisma 5](/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-5)
```

Comment:
Do these guides apply just the same if upgrading e.g. from Prisma 3 to Prisma 5? Or would a user need both the Prisma 4 and 5 upgrade guides?

Reply:
They would need to follow each one, we do not repeat all the previous instructions. So technically this is a "Prisma 4 to Prisma 5 upgrade guide" indeed.

Reply:
Should we make that more clear, or are we ok as-is? If we're ok, I'm good if anyone resolves this comment.

Reply:
I think for now we're good. I have some issues planned that will revamp our guides and will also link to them from here.